palm/plugins/core/commands/cmd_containerize.py

```diff
@@ -12,9 +12,9 @@
     default="3.8",
     help="Python version to use (default 3.8)",
 )
-@click.pass_context
-def cli(ctx, version: str):
+@click.pass_obj
+def cli(environment, version: str):
     all_templates_dir = Path(Path(__file__).parents[1], "templates")
     template_dir = all_templates_dir / "containerize"
-    PythonContainerizer(ctx, template_dir, version).run()
-    click.secho(f"Containerized {ctx.obj.palm.image_name}", fg="green")
+    PythonContainerizer(environment, template_dir, version).run()
```

Comment:
This will fail with the reverted changes in containerizer.py

Let's revert the changes in this file too and make a GH issue tagged with v3 (next major version) to update this!